catroid/src/main/java/org/catrobat/catroid/ui/ScratchProgramDetailsActivity.java

```diff
@@ -187,16 +157,18 @@ protected void onStart() {
 	@Override
 	protected void onDestroy() {
 		super.onDestroy();
-		Log.d(TAG, "Destroyed " + TAG);
 		conversionManager.removeJobViewListener(programData.getId(), this);
 		conversionManager.removeGlobalDownloadCallback(this);
 		fetchRemixesTask.cancel(true);
 		progressDialog.dismiss();
 	}
 
 	private void setUpActionBar() {
-		getSupportActionBar().setTitle(R.string.title_activity_scratch_converter);
-		getSupportActionBar().setHomeButtonEnabled(true);
+		setSupportActionBar((Toolbar) findViewById(R.id.toolbar));
```

Comment:
https://stackoverflow.com/questions/8999591/difference-between-getstring-and-getresources-getstring/8999657